server/lomas_server/tests/test_mongodb_admin.py

```diff
@@ -6,7 +6,7 @@
 from pymongo import MongoClient
 
 from admin_database.utils import get_mongodb_url
-from administration.mongodb_admin import (
+from mongodb_admin import (
```

Comment:
I wonder why the github action did not trigger any error for my PR (?) it is a bit worrying

Reply:
not sure, i think there is no issue? If you refer to the PR were you created this test, it was correct since the location of mongodb_admin.py was in another path.

Reply:
ah ok I did not see that it was moved !